ui/raidboss/data/06-ew/dungeon/lapis_manalis.txt

```diff
@@ -0,0 +1,283 @@
+# Lapis Manalis
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839::.*is no longer sealed/ window 100000 jump 0
+
+#~~~~~~~~#
+# Albion #
+#~~~~~~~~#
+
+# -ii 7A7D
+
+0.0 "--sync--" sync / 00:0839::The Silvan Throne will be sealed off/ window 0,1
+6.0 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+10.6 "Call of the Mountain" sync / 1[56]:[^:]*:Albion:7A7C:/
+21.6 "Wildlife Crossing 1" sync / 00:0839::A rumbling in the distance draws closer.../ window 30,30 duration 15
+35.6 "Wildlife Crossing 2" sync / 00:0839::More rumbling can be heard in the distance.../ window 30,30 duration 15
+39.2 "Albion's Embrace" sync / 1[56]:[^:]*:Albion:7A85:/
+45.3 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+51.7 "Left/Right Slam" sync / 1[56]:[^:]*:Albion:(802E|802D):/
+60.3 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7E:/
+62.3 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7F:/
+68.6 "Icebreaker" sync / 1[56]:[^:]*:Albion:7A81:/
+71.6 "Icy Throes" sync / 1[56]:[^:]*:Albion:7A82:/
+77.8 "Icy Throes" sync / 1[56]:[^:]*:Albion:(800F|7A83):/
+78.2 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+82.8 "Icy Throes" sync / 1[56]:[^:]*:Albion:7FB9:/
```

Comment:
Same questions here, re: Icy Throes.  This is logically just a single spread damage attack, right?  If so, then ideally just label it once in the timeline so it doesn't look like three things.  Usually I either pick the castbar (if it's something that people want to mit) or the damage itself (if there's a timing issue folks want to know about), and label the other abilities `--sync--` so they don't show up.

Reply:
There is a second attack here after the spread that is targeted at the environment, but it's just an aoe dodge check.  The first is just the boss castbar for the first spread/environment, I'll sync that one out.